docs/source/search/plugin.rst

```diff
@@ -0,0 +1,48 @@
+API Reference
+*************
+
+The :class:`IndicoSearchProvider` interface allows an abstract service to integrate with Indico's search module.
```

Comment:
s/abstract/external/? The service implementation isn't abstract anymore

Reply:
Sounds good, with abstract I meant any type of external service.